src/main/java/com/conveyal/datatools/manager/controllers/api/GtfsPlusController.java

```diff
@@ -55,30 +54,10 @@
     private static FeedStore gtfsPlusStore = new FeedStore("gtfsplus");
 
 
-    public static Boolean uploadGtfsPlusFile (Request req, Response res) throws IOException, ServletException {
-
-        //FeedSource s = FeedSource.retrieveById(req.queryParams("feedSourceId"));
+    private static Boolean uploadGtfsPlusFile (Request req, Response res) {
```

Comment:
Please add some comments to this method explaining what it does.  Also, please add some comments to the class in addition to the original author.